### ISSUE TITLE
500 error for partial reference

### ISSUE THREAD

**Issue by aaccomazzi** [adsabs/reference_service#31]
When using the following partial reference, the service returns a 500 error:
```
Accomazzi, A., Eichhorn, G., Kurtz, M.J., et al. 2000, A&AS
```
The corresponding API URL is `https://ui.adsabs.harvard.edu/v1/reference/text/Accomazzi%2C%20A.%2C%20Eichhorn%2C%20G.%2C%20Kurtz%2C%20M.J.%2C%20et%20al.%202000%2C%20A%26AS`

**Comment by golnazads** [comment]
sorry, there is something going on with caching that I am going to track down to see what is happening. But if I disable the caching this is what it come back as
2021-02-21T18:21:22.934Z 934 ERROR    [99721:Thread-1:views.py:151] Exception: ('Not enough information to resolve the record.', <referencesrv.resolver.hypotheses.Hypotheses object at 0x151ee7af0>)
it would not even attempt it when it knows it is journal that has to have more information. that is the right thing to do, right?

**Comment by aaccomazzi** [comment]
Correct.  We should catch the exception and return the error message to the user in a properly encoded json message.
